pkg/repositories/repositories.go

```diff
@@ -9,29 +9,30 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
-type describedObject struct {
-	Name        string
-	Description string
-}
-
 // Repository represents a codified description of a target component
 type Repository struct {
-	describedObject    `yaml:",inline"`
+	Name               string
+	Description        string
 	URL                string
 	CertificationLevel string `yaml:"certification,omitempty"`
 	Version            string `yaml:"version,omitempty"`
 	AfterVersion       string `yaml:"after,omitempty"`
 	UpgradeURL         string `yaml:"upgrade_url,omitempty"`
+	Tool               string `yaml:"tool,omitempty"`
```

Comment:
This field's name is extremely ambiguous

Reply:
I think we can remove this